src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
+
+    public Integer getQuantity(String fruit) {
+        return fruits.get(fruit);
+    }
+
+    public void put(String fruit, Integer quantity) {
+        fruits.put(fruit, quantity);
+    }
+
+    public Set<String> keyset() {

```

Comment:
The method 'keyset' returns a Set view of the keys contained in the map. Any modification to this set will affect the original map. Consider returning a copy of the set to maintain data integrity.

Reply:
ok, done